<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewtestdetails</class>
 <widget class="QDialog" name="viewtestdetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test Details</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/new/img/Img/logo.jpg</normaloff>:/new/img/Img/logo.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(161, 235, 76);</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>160</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>16</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string>Select Student ID :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>640</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Click here to cancel</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(141, 141, 141);
border-radius:20%;


font: 75 16pt &quot;MS Shell Dlg 2&quot;;
font: 75 16pt &quot;Rockwell&quot;;</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>16</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string>First Name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>16</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string>Last Name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>16</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string>Correct Questions :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>340</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>16</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string>Wrong Questions :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>390</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>16</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string>Company:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>200</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_lname">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>240</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_correct">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>290</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_wrong">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>340</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_score">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>490</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gotohome">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>670</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Click here to go to Home</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(142, 173, 200);
border-radius:20%;

font: 75 22pt &quot;Rockwell&quot;;
font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Go To Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_load">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>640</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Click to search Student ID</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(141, 141, 141);
border-radius:20%;


font: 75 16pt &quot;MS Shell Dlg 2&quot;;
font: 75 16pt &quot;Rockwell&quot;;</string>
   </property>
   <property name="text">
    <string>Load ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>440</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>16</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string>Job Profile :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_job">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>430</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>490</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>16</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string>Score Obtained :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_company">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>390</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>550</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>16</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string>Result Status:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_status">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>540</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>50</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Click here to Logout</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/img/Img/logout.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>461</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gadugi</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>STUDENT TEST DETAILS</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
